src/main/java/lc/captchas/PoppingCharactersCaptcha.java

```diff
@@ -63,21 +65,32 @@ private int jitter() {
       final var writer = new GifSequenceWriter(output, 1, 900, true);
       final var advances = computeOffsets(text);
       final var prevColor = Color.getHSBColor(0f, 0f, 0.1f);
-      IntStream.range(0, text.length()).forEach(i -> {
-        final var color = Color.getHSBColor(HelperFunctions.randomNumber(0, 100)/100.0f, 0.6f, 1.0f);
-        final var prevI = (i - 1 + text.length()) % text.length();
-        final var nextImage = makeImage((g) -> {
-          g.setColor(prevColor);
-          g.drawString(String.valueOf(text.charAt(prevI)), advances[prevI] + jitter(), 45 + jitter());
-          g.setColor(color);
-          g.drawString(String.valueOf(text.charAt(i)), advances[i] + jitter(), 45 + jitter());
-        });
-        try {
-          writer.writeToSequence(nextImage);
-        } catch (final IOException e) {
-          e.printStackTrace();
-        }
-      });
+      IntStream.range(0, text.length())
+          .forEach(
+              i -> {
+                final var color =
+                    Color.getHSBColor(HelperFunctions.randomNumber(0, 100) / 100.0f, 0.6f, 1.0f);
+                final var prevI = (i - 1 + text.length()) % text.length();
+                final var nextImage =
+                    makeImage(
+                        (g) -> {
+                          g.setColor(prevColor);
+                          g.drawString(
+                              String.valueOf(text.charAt(prevI)),
+                              advances[prevI] + jitter(),
+                              45 + jitter());
+                          g.setColor(color);
+                          g.drawString(
+                              String.valueOf(text.charAt(i)),
+                              advances[i] + jitter(),
+                              45 + jitter());
+                        });
+                try {
+                  writer.writeToSequence(nextImage);
+                } catch (final IOException e) {
+                  e.printStackTrace();
+                }
+              });
```

Comment:
This looks very verbose. Is it because the line length is set to a small number? I think we had changed it for Scala. Can we set the same for Java?

Reply:
> This plugin uses the Google Java Format library, which makes it quite opinionated and not particularly configurable.
> If you want to tweak the format, take a minute to consider whether it is really worth it, and have a look at the motivations in the Google Java Style Guide. If you decide you really need more flexibility, you could consider other plugins such as the [sbt-checkstyle-plugin](https://github.com/etsy/sbt-checkstyle-plugin)

Found this in the Readme. We'll have to change the formatter.

Reply:
Oh, let's not change the formatter. If the line length can be changed easily with the current formatter, then we can do that, or else, live with the (slight) annoyance.

I think I need to get into the habit of running a sbt compile before committing. I was only compiling with vscode earlier.

Also, we seem to be using a very old version of scalafmt (2.4.2). The latest is 2.7.5

Reply:
There's seems to be some conflict between scalafmt version and sbt version. We'll need to upgrade both based on compatibility.